cincinnati/src/plugins/internal/channel_filter.rs

```diff
@@ -41,6 +42,20 @@ impl ChannelFilterPlugin {
 
         Ok(Box::new(plugin))
     }
+
+    pub fn remove_conditional_edges(
+        &self,
+        ce: &mut Vec<cincinnati::ConditionalEdge>,
+        releases: HashSet<String>,
+    ) -> Fallible<()> {
+        ce.into_iter()
```

Comment:
I think this function and it's arch-filter analog need a separate layer somewhere to completely remove `ConditionalEdge` entries if edge-filtering remove all of their edges.

Reply:
handled this case